### ISSUE TITLE
[generate:entity:content] Need to call the parent constructor

### ISSUE THREAD

**Issue by adrian-cid** [hechoendrupal/drupal-console#3971]
### Problem/Motivation
Once you create an entity and go to edit it you will see this error:

`Error: Call to a member function getTranslationFromContext() on null in Drupal\Core\Entity\ContentEntityForm->initFormLangcodes() (line 313 of /home/dmriq/www/core/lib/Drupal/Core/Entity/ContentEntityForm.php) #0 `

Is the same error on:

[https://www.drupal.org/project/votingapi_widgets/issues/2996444](https://www.drupal.org/project/votingapi_widgets/issues/2996444)
[https://www.drupal.org/project/mailchimp/issues/2999528](https://www.drupal.org/project/mailchimp/issues/2999528)

### How to reproduce
Drupal version 8.5.2
Console version 1.8

Steps to reproduce
generate an entity via the drupal console with the generate:entity:content command

Go to create an entity and then go to edit it.

### Solution
And it seems to be related with Drupal 8.6, I resolve the problem injecting the dependencies and calling the parent constructor like this patch:

[https://www.drupal.org/files/issues/2018-08-31/2996444.patch](https://www.drupal.org/files/issues/2018-08-31/2996444.patch)

**Comment by joaquinbravo-natgeo** [comment]
Just tried to reproduce the error and got nothing. Using drupal 8.6.1. Could you maybe provide more details? Maybe some of the options of the entity you created... maybe it has more fields than just the default ones?

This was the complete command I used to generate the entity:

```$ drupal generate:entity:content  --module="example" --entity-class="DefaultEntity" --entity-name="default_entity" --base-path="/admin/structure" --label="Default entity" --is-translatable --revisionable --learning --uri="http://default" --no-interaction```

Then I was able to create and edit entities.

**Comment by adrian-cid** [comment]
I remember now why I have the problem.

I create the entity without bundles, without translation and revisionable.

The code is created and works fine, but you don't inject the current_user service inside the entity edit form as you have this call `$entity->setRevisionUserId(\Drupal::currentUser()->id());` , so once I check the coding standard I see this problem and I injected the current_user service, and then I got the described error.

So, maybe we need to rename this issue?
